cmd/destroy.go

```diff
@@ -16,54 +21,51 @@ var destroyCmd = &cobra.Command{
 	Use:     "destroy",
 	Short:   "destroy a lab",
 	Aliases: []string{"des"},
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		c := clab.NewContainerLab(debug)
 		err := c.Init(timeout)
 		if err != nil {
-			log.Fatal(err)
+			return err
 		}
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
-		if prefix != "" {
```

Comment:
you removed the prefix based removal
does it mean that you see that destroy should act on the topo file only?
In that case we need to remove the `-p/--prefix` flag from cmd

Reply:
yes, it doesn't work properly, I was hoping to fix it in another PR.
The flag can be removed